### ISSUE TITLE
Use case: Filtered Index Support

### ISSUE THREAD

**Issue by Grauenwolf** [Grauenwolf/DotNet-ORM-Cookbook#113]
The use of filtered indexes requires the value not be passed as a normal parameter.

Dapper: https://github.com/StackExchange/Dapper#literal-replacements

**Comment by FransBouma** [comment]
Not sure this is a scenario we'd want, as it underlines literals in the query instead of the usage of parameters.

**Comment by Grauenwolf** [comment]
The goal isn't to demonstrate literals in a query, just the ability to use the filtered index. 

So I'll include a database view with the filter already in place.

```
CREATE VIEW ActiveFoo As 
SELECT * FROM Foo WHERE IsActive = 1 
```

**Comment by FransBouma** [comment]
I'm not sure I fully understand the scenario then :) To use the filtered index, one has to emit the literal in the query, correct? Or do you mean: if you can't do that, use the view ?

**Comment by Grauenwolf** [comment]
Correct. Using a literal or using a view would both have the desired effect of using the filtered index.
